practical/bash/import_as_create_annotation.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+#   Copyright (C) 2018 University of Dundee. All rights reserved.
+#
+#   This program is free software; you can redistribute it and/or modify
+#   it under the terms of the GNU General Public License as published by
+#   the Free Software Foundation; either version 2 of the License, or
+#   (at your option) any later version.
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public License along
+#   with this program; if not, write to the Free Software Foundation, Inc.,
+#   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# ------------------------------------------------------------------------------
+#
+# In this script we:
+#   * log in as another user
+#   * create a new dataset
+#   * import an image
+#   * upload a attachment
+#
+#
+dir=$(pwd -P)
+OMEROPATH=${OMEROPATH:-/opt/omero/server/OMERO.server/bin/omero}
+PASSWORD=${PASSWORD:-ome}
+HOST=${HOST:-outreach.openmicroscopy.org}
+SUDOER=${SUDOER:-trainer-1}
+OMEUSER=${OMEUSER:-user-1}
+IMAGEPATH=${IMAGEPATH:-/}
+ATTACHMENTPATH=${ATTACHMENTPATH:-/}
+
+deleteimage=false
+deletefile=false
+
+if [ "$IMAGEPATH"="/" ]; then
+    touch image_to_import.fake
+    IMAGEPATH=$dir/image_to_import.fake
+    deleteimage=true
+fi
+#printf $IMAGEPATH
```

Comment:
do you want to retain this line for further use or is it just a testing remnant ?